Source/DataAccessLayer/Bases/NetTiersProviderBase.cst

```diff
@@ -93,7 +100,36 @@ namespace <%=DALNameSpace%>.Bases
                 }
                 if (entityCreationalFactoryType == null)
                     throw new ArgumentNullException("Could not find a valid entity factory configured in assemblies.  .netTiers can not continue.");
-                
+
+                /*
+				 * Attempt to create the Exception Handler
+				 */
+                if (!string.IsNullOrEmpty(exceptionHandlerTypeString))
+                {
+                    //exceptionHandlerType = Type.GetType(exceptionHandlerTypeString, false, true);
+
+                    if (exceptionHandlerType == null && exceptionHandlerTypeString.Contains(","))
+                    {
+                        //assembly still not found, try loading the assembly.  It's possible it's not referenced.
+                        string exceptionHandlerTypeStringAssembly = exceptionHandlerTypeString.Substring(exceptionHandlerTypeString.IndexOf(',') + 1).Trim();
+
+                        try
+                        {
+                            System.Reflection.Assembly.Load(exceptionHandlerTypeStringAssembly);
+                            exceptionHandlerType = Type.GetType(exceptionHandlerTypeString, false, true);
+                        }
+                        catch
+                        {
+                            //throws file not found exception
+                        }

```

Comment:
I never like just eating exceptions. Can we log it, or just handle the file not found exception?